Makefile

```diff
@@ -57,8 +57,16 @@ endif
 
 GENERAL_TAGS := 'include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo libbpf '
 GPU_TAGS := ' gpu '
```

Comment:
Note: These GPU build tags mean that we are filtering out GPU implementations from the base Kepler image. 

This places a requirement that the Kepler operator pick the right image for the accelerators supported on the node where Kepler is deployed.

The reasoning behind excluding the GPU specific implementations from the base Kepler image is that DCGM grows the image size by 2GB...

 We can continue this trend of a separate image per accelerator, or perhaps we can just combine all the accelerator builds into a single image (as this is going to be large anyway)... OR
 
 we just build everything into the default Kepler image (removing that requirement from the operator to pick and deploy specific images)